.editorconfig

```diff
@@ -10,5 +10,5 @@ trim_trailing_whitespace = true
 [*.md]
 trim_trailing_whitespace = false
 
-[{*.json,.eslintrc,.babelrc}]
-indent_size = 2
\ No newline at end of file
+[*.json]
```

Comment:
We should probably reindent the `.eslintrc` file to make sure it is following the standard.